segments/__init__.py

```diff
@@ -18,6 +20,15 @@ def __init__(self, *args, **kwargs):
             # Other segments are active if the config files states so.
             self.active = config.SEGMENTS.get(class_name, False)
 
+        if( config.SPACES ):
+            if class_name not in ['newline', 'root', 'divider', 'padding', 'exitcode']:
```

Comment:
for ```ExitCode``` there are already two spaces
```
        self.text = ' ' + glyphs.CROSS + ' '
```
I don't like how it appear with more. 

---

For the ```Root``` I think it's better this way so it seam a normal prompt line.
I think it's a matter of preference.